spec.html

```diff
@@ -43572,7 +43572,7 @@ <h1>PromiseCapability Records</h1>
                 [[Promise]]
               </td>
               <td>
-                an Object
+                an Object or *undefined*
```

Comment:
This `*undefined*` is never observable and only used as a placeholder during construction. I'd rather change the way we construct the PromiseCapability Record in `NewPromiseCapability`. We can use a different kind of record for collecting the resolve/reject functions (or no record at all if we're comfortable assigning to closed-over aliases) and then construct a proper PromiseCapability Record at the end.